App/Components/_colors.js

```diff
@@ -0,0 +1,13 @@
+const colors = {
```

Comment:
I mentioned this in the meeting last week, but we can give these colors meaningful names? There are quite a few conventions out there. 
This article has a pretty good guide on how to name colors:  https://davidwalsh.name/sass-color-variables-dont-suck
another good resource, that I look at all the time for naming style elements is bootstrap's sass variables: https://github.com/twbs/bootstrap-sass/blob/master/assets%2Fstylesheets%2Fbootstrap%2F_variables.scss
Take a look at their grays line 11 - 16.